ui/readme.de.md

```diff
@@ -94,6 +94,13 @@ Manchmal ist es jedoch notwendig, einen bestimmten Query zu finden, der exakt de
 z.B.: **+hemd blau xl**.  
 Diese exakte Suchen liefert einen einzelnen Cluster zurück, wenn eine darin enthaltene Variante genau mit dem eingegebenen Query übereinstimmt. 
 
+# Redirects
```

Comment:
Please use German headline. Use the same wording as the UI in the left-hand side menu. Add `{redirects}` at the end.

Reply:
fixed. Also fixed it for other headlines in the Enhance section. Obviously we never used the same wording as in the UI in de and fr.